### PR TITLE
Allow resizing of timeline chart

### PR DESCRIPTION
**Pull request by abhinava-ericsson**

### CODE REVIEW

File: timeline-chart/src/time-graph-container.ts [hunk 1]
```diff
@@ -75,9 +75,10 @@ export class TimeGraphContainer {
     }
 
     // if canvas size has changed displayWidth need to be updated for zoomfactor
-    reInitCanvasSize(newWidth: number){
-        this.application.renderer.resize(newWidth, this.config.height);
+    reInitCanvasSize(newHeight: number, newWidth: number){
```
Comment (tahini) [inline timeline-chart/src/time-graph-container.ts:78]: For backward-compatibility, you should put newHeight as second parameter

File: timeline-chart/src/time-graph-state-controller.ts [hunk 2]
```diff
@@ -24,8 +24,8 @@ export class TimeGraphStateController {
 
     constructor(protected canvas: HTMLCanvasElement, protected unitController: TimeGraphUnitController) {
```
Comment (tahini) [inline timeline-chart/src/time-graph-state-controller.ts:25]: TIL: is this really a way to assign a new field directly from the constructor's signature?? just putting protected makes this.canvas available?

File: timeline-chart/src/time-graph-state-controller.ts [hunk 3]
```diff
@@ -72,6 +72,10 @@ export class TimeGraphStateController {
         return this._canvasDisplayHeight;
     }
 
+    updateDisplayHeight() {
+        this._canvasDisplayHeight = this.canvas.height/ this.ratio;
```
Comment (tahini) [inline timeline-chart/src/time-graph-state-controller.ts:76]: Add white space before /

**Review by tahini — CHANGES_REQUESTED** [review]
Small comment, but works well! nice improvement

File: timeline-chart/src/time-graph-container.ts (hunk 1 above)
Comment (tahini) [reply timeline-chart/src/time-graph-container.ts:78]: Also, for backward-compatibility, make the second argument optional with an ok default value

File: timeline-chart/src/time-graph-state-controller.ts (hunk 2 above)
Comment (abhinava-ericsson) [reply timeline-chart/src/time-graph-state-controller.ts:25]: I found that weird too. Assumed that it is canonical Javascript.

File: timeline-chart/src/time-graph-container.ts (hunk 1 above)
Comment (bhufmann) [reply timeline-chart/src/time-graph-container.ts:78]: Also add a space before {

**Review by bhufmann — CHANGES_REQUESTED** [review]

**Comment by abhinava-ericsson** [comment]
@tahini @bhufmann Added a new patch. Please review.

File: timeline-chart/src/time-graph-container.ts [hunk 4]
```diff
@@ -75,9 +75,13 @@ export class TimeGraphContainer {
     }
 
     // if canvas size has changed displayWidth need to be updated for zoomfactor
-    reInitCanvasSize(newWidth: number){
-        this.application.renderer.resize(newWidth, this.config.height);
+    reInitCanvasSize(newWidth: number, newHeight?: number) {
```
Comment (tahini) [inline timeline-chart/src/time-graph-container.ts:78]: You can initialize from the method's signature like this: newHeight: number = this.config.height

**Review by tahini — APPROVED** [review]
Simple comment, but looks good

**Review by bhufmann — APPROVED** [review]
It looks good to me

**Review by bhufmann — CHANGES_REQUESTED** [review]
Please add the signed off to the commit message.

**Comment by abhinava-ericsson** [comment]
> Please add the signed off to the commit message.

I rebased to the latest master and signed-off. Please review and merge.

**Review by bhufmann — CHANGES_REQUESTED** [review]
Please use your name in the signed-of-by instead

**Review by bhufmann — APPROVED** [review]
Looks good to me. Thank you!

**Pull request merged**